source/sync/app-builder/local-to-sync.txt

```diff
@@ -12,34 +12,30 @@ Add Sync to a Local-Only App
    :depth: 2
    :class: singlecol
 
-Overview
---------
-
 If you have a mobile app that uses a local realm and you want to 
-convert the app to use Sync, there are a few steps you 
-need to follow. You first need to set up an App
-backend. Then, make code changes in your client applications.
-
-In the following sections, you will learn to do the following on the backend:
+convert the app to use Sync, there are two main tasks: create and configure 
```

Comment:
Because this is the first mention, we should probably use the full name.
```suggestion
convert the app to use Atlas Device Sync, there are two main tasks: create and configure 
```